### ISSUE TITLE
Imports dashboard

### ISSUE THREAD

**Issue by gdower** [CatalogueOfLife/clearinghouse-ui#652]
The imports queue is currently displaying running jobs in the finished queue which is a bug but I think it's actually an improvement having the running jobs and finished jobs combined onto a single imports dashboard page because it's faster to just check 1 page to determine an import job's state, plus we could reduce the number of menu items down from 3 (imports, running, finished) to 1 (imports) which will free up valuable sidebar space. If you can easily put multiple tables on a single page displaying the import jobs by their state, here's a suggest mock-up:

![imports mock-up](https://user-images.githubusercontent.com/8573609/84948456-62b5c100-b0b1-11ea-8b82-cd98d688d216.png)

**Comment by gdower** [comment]
The mock-up is editable here if you have suggestions: https://awwapp.com/b/uxl01nbl86nby/

**Comment by mdoering** [comment]
Would you not need scrollable tables in this design? I would strongly plea to avoid that anywhere in the UI. But maybe thats not needed if all 3 boxes are pageable on their own.

**Comment by gdower** [comment]
I agree with avoiding scrollable tables. I think it could just display the top 5 - 10 results with paging. Usually we'd be most interested to quickly see the status of the results that are sorted to the top, and paging would still allow us to dig through past imports for to debug the failed imports.
